asset/snippets/test/samples.test.js

```diff
@@ -0,0 +1,262 @@
+// Copyright 2021 Google LLC
```

Comment:
Did you mean to include this file?

Reply:
This singular test file includes not just the quickstart tests, but tests for all the other samples, as well. They are run sequentially.

I included it in the migration, so that when the language maintainers address the GH issue I created (https://github.com/GoogleCloudPlatform/nodejs-docs-samples/issues/2864), they would have the body of the individual tests already and could use their best judgment to modify them as needed to decompose into separate test files. At that point, I imagine the `samples.test.js` will be deleted.

I figure inclusion of this file here is acceptable tech debt for the migration. Since @sofisl already discarded most of the tests for her [migration to google-cloud-node](https://github.com/googleapis/google-cloud-node/blob/main/packages/google-cloud-asset/samples/test/sample.test.js), I didn't want to lose it in the migration/archival process.

Reply:
To address your other comment about migration destination:
i (and I believe most others) are following the example set in the migration guide, written by @grayside. You can see here that the samples end up `<product>/snippets`:
https://docs.google.com/document/d/1YhpsM5ylBFDunRsd-Vq7pHRNUz3uh9TUbOkJOZwCmOk/edit?resourcekey=0--XOS_x8NmzOS8_Qws-Qsug#bookmark=kix.z1viwa6f4215